tabular/src/autogluon/tabular/models/tabular_nn/torch/torch_network_modules.py

```diff
@@ -154,12 +154,15 @@ def init_params(self):
 
     def forward(self, data_batch):
         input_data = []
+        input_offset = 0
         if self.has_vector_features:
-            input_data.append(data_batch['vector'].to(self.device))
+            input_data.append(data_batch[0].to(self.device))
+            input_offset += 1
         if self.has_embed_features:
-            embed_data = data_batch['embed']
+            embed_data = data_batch[input_offset]
             for i in range(len(self.embed_blocks)):
                 input_data.append(self.embed_blocks[i](embed_data[i].to(self.device)))
+            input_offset += 1
```

Comment:
Why input_offset += 1 here? Seems like last time used?

Reply:
Nice catch. This is not necessary here.

Reply:
Removed.